recommenders/datasets/split_utils.py

```diff
@@ -163,3 +163,35 @@ def split_pandas_data_with_ratios(data, ratios, seed=42, shuffle=False):
         splits[i]["split_index"] = i
 
     return splits
+
+
+def filter_k_core(data, core_num=0, col_user="userID", col_item="itemID"):
+    """Filter rating dataframe for minimum number of users and items by
+    repeatedly applying min_rating_filter until the condition is satisfied.
+
+    """
+    num_users, num_items = len(data[col_user].unique()), len(data[col_item].unique())
+    print(f"Original: {num_users} users and {num_items} items")
```

Comment:
minor point can we add
```
import logging
log = logging.getLogger(__name__)
```
as in https://github.com/microsoft/recommenders/blob/main/recommenders/datasets/download_utils.py#L15
and then change the prints with log.info?

Reply:
Done.